cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -712,12 +709,16 @@ instance ToObject SlotNo where
     mkObject [ "kind" .= String "SlotNo"
              , "slot" .= toJSON (unSlotNo slot) ]
 
-
-instance ConvertRawHash header => ToObject (TraceFetchClientState header) where
+instance (HasHeader header, ConvertRawHash header)
+ => ToObject (TraceFetchClientState header) where
   toObject _verb BlockFetch.AddedFetchRequest {} =
     mkObject [ "kind" .= String "AddedFetchRequest" ]
   toObject _verb BlockFetch.AcknowledgedFetchRequest {} =
     mkObject [ "kind" .= String "AcknowledgedFetchRequest" ]
+  toObject _verb (BlockFetch.SendFetchRequest af) =
+    mkObject [ "kind" .= String "SendFetchRequest"
+             , "head"   .= String (renderChainHash (renderHeaderHash (Proxy @header)) $ AF.headHash af)
```

Comment:
Could you try to fit this into 80 columns?

Reply:
There are also some unnecessary spaces  here: `, "head"   .= String `

Reply:
Fixed.